pocs/focuser/focuslynx.py

```diff
@@ -294,19 +295,36 @@ def _send_command(self, command_str, expected_reply):
         Utility function that handles the common aspects of sending commands and
         parsing responses.
         """
+        name = self.port
+        if hasattr(self, '_focuser_config'):
+            name = self.uid
+
         # Make sure we start with a clean slate
         self._serial_port.reset_output_buffer()
         self._serial_port.reset_input_buffer()
         # Send command
         self._serial_port.write(command_str.encode('ascii'))
-        # Should always get '!' back unless there's an error
         response = str(self._serial_port.readline(), encoding='ascii').strip()
-        if response != '!':
-            message = "Error sending command '{}' to {}: {}".format(command_str, self.uid, response)
+        if not response:
+            message = "No response to command '{}' from {}".format(command_str, name)
```

Comment:
Maybe the error messages here could include the literal string "focuser", i.e. "No response to command 'foo' from focuser 'bar'"